### PR TITLE
Minor changes to deploy unittest

### PR DESCRIPTION
**Pull request by amaltaro**
Minor changes:
- change WMA_TAG in case you want another version
- don't clone using ssh key
  @ticoann please review

### CODE REVIEW

File: test/deploy/deploy_unittest.sh [hunk 1]
```diff
@@ -1,13 +1,17 @@
 #!/bin/bash
-git clone git@github.com:dmwm/deployment.git
-(cd deployment; curl https://github.com/dmwm/deployment/commit/3309be9ce3b628efe39e1db30b6a1f865e1fa3e4.patch | git am)
+WMA_TAG=1.0.7.pre3
```
Comment (ericvaandering) [inline test/deploy/deploy_unittest.sh]: Hey, as long as you are doing this (I do something similar) why don't you pick up an externally set variable if it's set?

File: test/deploy/deploy_unittest.sh [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
+git clone https://github.com/dmwm/deployment.git
```
Comment (ericvaandering) [inline test/deploy/deploy_unittest.sh]: dmwm should be a configurable variable too. I want to check out my fork so I can make changes.

File: test/deploy/deploy_unittest.sh [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 curl -s https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/init.sh > init.sh
 curl -s https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/env_unittest.sh > env_unittest.sh
 curl -s https://raw.githubusercontent.com/dmwm/WMCore/master/test/deploy/WMAgent_unittest.secrets > WMAgent_unittest.secrets
 source ./init.sh
-$PWD/deployment/Deploy -R wmagent-dev@1.0.6.pre7 -r comp=comp.pre -t 1.0.6.pre7 -A slc6_amd64_gcc481 -s 'prep sw post' $INSTALL_DIR admin/devtools wmagent
-(mkdir $TEST_DIR; cd $TEST_DIR; git clone git@github.com:dmwm/WMCore.git)
+echo ""
+$PWD/deployment/Deploy -R wmagent-dev@$WMA_TAG -r comp=comp.pre -t $WMA_TAG -A slc6_amd64_gcc481 -s 'prep sw post' $INSTALL_DIR admin/devtools wmagent
+echo ""
+(mkdir $TEST_DIR; cd $TEST_DIR; git clone https://github.com/dmwm/WMCore.git)
```
Comment (ericvaandering) [inline test/deploy/deploy_unittest.sh]: Or configurable fork here. Don't know which.

File: test/deploy/deploy_unittest.sh (hunk 1 above)
Comment (amaltaro) [inline test/deploy/deploy_unittest.sh]: How about providing it in the command line then?

File: test/deploy/deploy_unittest.sh (hunk 2 above)
Comment (amaltaro) [inline test/deploy/deploy_unittest.sh]: Command line too?

Comment (ericvaandering) [inline test/deploy/deploy_unittest.sh]: command line options would be fine.

> On May 13, 2015, at 22:17, Alan Malta Rodrigues <notifications@github.com> wrote:
> 
> In test/deploy/deploy_unittest.sh <https://github.com/dmwm/WMCore/pull/5902#discussion_r30271245>:
> 
> > @@ -1,13 +1,17 @@
> >  #!/bin/bash
> > -git clone git@github.com:dmwm/deployment.git
> > -(cd deployment; curl https://github.com/dmwm/deployment/commit/3309be9ce3b628efe39e1db30b6a1f865e1fa3e4.patch | git am)
> > +WMA_TAG=1.0.7.pre3
> > +git clone https://github.com/dmwm/deployment.git
> Command line too?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/dmwm/WMCore/pull/5902/files#r30271245>.
> 




Eric W. Vaandering 
Computing Physics Developer
CMS Computing Development Lead

Scientific Computing Division
Fermi National Accelerator Laboratory
630 962 7260 office
www.fnal.gov <http://www.fnal.gov/>

**Comment by ticoann** [comment]
Alan, Eric, I just wanted to have a quick version for the unittest now. :smile:
All the suggestions are valid.
1. WMA_TAG=1.0.7.pre3  - this should have command line option for different version (not that important as long as library changes, since the code itself will be overwritten)
2. cloning source code  - with option to clone form different sources as Eric said, this would be more important.
3. able to have option to use MySQL or Oracle as DB backend. (default with MySQL - I will try to set up db without passing the password - obviously for Oracle, we can't do that)

I will update the script first with 1. 2.

**Comment by ticoann** [comment]
I updated the script with 1. 2. option above
#5903

**Comment by amaltaro** [comment]
Closing it without merging in favor of #5903

**Pull request closed without merging**